openwisp_users/tests/test_admin.py

```diff
@@ -375,3 +375,44 @@ def test_change_user(self):
         self.assertNotContains(response, 'error')
         user.refresh_from_db()
         self.assertEqual(user.bio, params['bio'])
+
+
+class TestMultitenantAdmin(TestMultitenantAdminMixin, TestOrganizationMixin):
+
```

Comment:
please remove this blank line

Reply:
Shouldn't there be one blank line before a method and two blank lines before a glocal class declaration?